silq/tools/plot_tools.py

```diff
@@ -791,10 +796,12 @@ class DCSweepPlot(ScanningPlot):
     """
     gate_mapping = {}
     point_color = 'r'
+
     # DCSweepParameter type
     def __init__(self,
                  parameter: _BaseParameter,
                  gate_mapping: Dict[str, str] = None,
+                 buffer_length: int = 1,
```

Comment:
I don't think buffer_length is the best term for this. If the purpose is to average over multiple shots, I would call it something like "averages". The reason is that buffer_length can also be interpreted as "the length of a buffer", i.e. the number of trace points in a single buffer.

It would also be useful to have a docstring with the description of what this does

Reply:
I see, ok, can rename